src/app/AddProductForm.js

```diff
@@ -229,7 +250,7 @@ export default function AddProductForm() {
       <SelectField
         label="Categoria"
         name="categoria"
-        value={formData.categoria}
+        value={formData.category}
         onChange={handleCategoriaChange}
         options={["Seleccione una categoría", "Women", "Men", "Boy", "Girl"]}
```

Comment:
Change it to English

Reply:
The suggestion is not possible due to it is already in English, please review the code carefully.